.github/workflows/oss_unencrypted.yml

```diff
@@ -0,0 +1,94 @@
+name: OSS ITs no encryption
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.6
+    - name: Compress the collector
+      run: |
+          tar czf diagnostic-collection.tar.gz ./*
+    - uses: webfactory/ssh-agent@v0.4.1
+      with:
+          ssh-private-key: ${{ secrets.TLP_CLUSTER_KEY }}
+    - name: Install dependencies
+      run: |
+          python -m venv venv
+          . venv/bin/activate
+          python -m pip install --upgrade pip
+          pip install behave
+          sudo apt-get update
+          sudo apt-get install openjdk-8-jre -y
```

Comment:
Question: Lately I saw there were some regressions introduced with 275 build of openjdk8. Does that apply here at all? If so, do we need to stick a particular build number?

Reply:
This u275 issue will affect long running clusters, leaving open sockets piling up.
I don't think that'll affect the tests we're doing here.

If such an issue hit us, we can definitely lock ourselves on a specific JDK version, but for now I guess it's good to test the latest available.